### PR TITLE
As a signage user, I'd like the default slide duration to be 10 seconds.

### PR DESCRIPTION
**Pull request by tatwell**
https://trello.com/c/rfpMnYle/92-as-a-signage-user-i-d-like-the-default-slide-duration-to-be-10-seconds

### CODE REVIEW

File: db/migrate/20161025224137_remove_duration_default_value_from_slides.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class RemoveDurationDefaultValueFromSlides < ActiveRecord::Migration
+  # Default will not be specified in model.
```
Comment (homeofmatt) [inline db/migrate/20161025224137_remove_duration_default_value_from_slides.rb]: Did you mean `Default will be specified in model`?

Comment (tatwell) [inline db/migrate/20161025224137_remove_duration_default_value_from_slides.rb]: I did. I will fix.

**Comment by homeofmatt** [comment]
Normally I'd be against setting defaults for attributes in this manner, but for this situation this makes a lot of sense. Might be worth just ditching the `duration` field from the slides in general if we're going to have a constant duration across all slides. We could just set that in one place rather than checking each slide for its duration value.

**Comment by tatwell** [comment]
Product owner wants users to still be able to change the duration. He just wants the default to be shorter. I moved it to model because I figure it'd be easier to update in future should they change their minds again.

**Comment by homeofmatt** [comment]
Yea that's what I meant about it making sense in this situation. The default is easy to update this way.
When you say "the product owner wants users to still be able to change the duration", do you mean the default duration across all slides or the duration per slide?
If you meant per slide then yes. Let's just keep it all as is. If you meant the default duration across all slides then that's where I meant we may just want to drop the `duration` field from slides if they're never going to have varying durations per slide.

**Pull request merged**